### PR TITLE
Read csv from log

### PR DESCRIPTION
**Pull request by nonifier**
Rework the code so that the tool can take a input file with a loosy format.
the new format is composed of 2 fields.
1 - `path` : the pURI path to the HTTP ressource (eg. /v1/coverage/sandbox/status)
2 - `parameters` : the GET parameters to be added to the HTTP request.

### CODE REVIEW

File: jormungandr_bench/bench.py [hunk 1]
```diff
@@ -179,24 +76,63 @@ def plot_normalized_box(array1, array2, label1='', label2=''):
...
+class Scenario(object):
+    def __init__(self, name, output_dir):
+        self.name = name
+        self.times = []
+        
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+
+        self.output = open(os.path.join(output_dir, "{}.csv".format(name)), 'w')
+        print("No,url,elapsed time,status_code", file=self.output)
```
Comment (xlqian) [inline jormungandr_bench/bench.py]: no close? better use a `with open(os.path.join(output_dir, "{}.csv".format(name)), 'w') as f`

File: jormungandr_bench/bench.py [hunk 2]
```diff
@@ -179,24 +76,63 @@ def plot_normalized_box(array1, array2, label1='', label2=''):
...
+        futures = (f for f in submit_work(pool, input_file, scenarios, extra_args))
+
+        for i in tqdm.tqdm(as_completed(futures), total=input_file_size*2):
+            idx, scenario_name, status_code, time, url = i.result()
+            
+            if status_code >= 300:
+                print('error ', status_code, ' occurred on the url: ', url)
+                continue
+
+            print("{},{},{},{}".format(idx, url, time, status_code), file=scenarios[scenario_name].output)
```
Comment (xlqian) [inline jormungandr_bench/bench.py]: hmmm, the opened file is never closed :)

File: jormungandr_bench/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ pip3 install -r requirements.txt
 Then do :
 
 ```bash
-./bench.py bench --input=benchmark_example.csv -a 'first_section_mode[]=car&last_section_mode[]=car'
+NAVITIA_API_URL=http://api.navitia.io TOKEN="3b036afe-0110-4202-b9ed-99718476c2e0" python3 bench.py bench -i ./benchmark_example.csv -c 6
```
[the remark this answers is not in the document]
Comment (nonifier) [reply jormungandr_bench/README.md]: that's the one in for Sandbox coverage. It's public in our documentation already ;-)
http://doc.navitia.io/#third-step

File: jormungandr_bench/bench.py [hunk 4]
```diff
@@ -179,24 +76,70 @@ def plot_normalized_box(array1, array2, label1='', label2=''):
...
+        self.name = name
+        self.times = []
+        
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+
+        self.output = open(os.path.join(output_dir, "{}.csv".format(name)), 'w')
+        print("No,url,elapsed time,status_code", file=self.output)
+
+    def __del__(self):
```
Comment (xlqian) [inline jormungandr_bench/bench.py]: OK, it's a C++ way :D the pythonic way is use `contextmanager`, never mind

**Pull request merged**